test/unit/SupSub.test.js

```diff
@@ -158,10 +158,10 @@ suite('SupSub', function() {
       assert.equal(mq.latex(), 'x_a^b');
 
       mq.keystroke('Left Backspace');
-      assert.equal(mq.latex(), 'xb_a');
+      assert.equal(mq.latex(), 'x_ab');
```

Comment:
it doesn't move to the subscript, we just output LaTeX with all unnecessary spaces removed even if they improve usability; that LaTeX is equivalent to `x_a b` or `x_{a}b`. There's an open issue somewhere to output `x_a b` instead, which I agree with, but I'm not quite sure how